source/_integrations/systemmonitor.markdown

```diff
@@ -18,13 +18,43 @@ To add this platform to your installation,
 add the following to your `configuration.yaml` file:
 
 ```yaml
-# Example configuration.yaml entry
-sensor:
+# Example configuration.yaml entry with all entry types (delete/comment out as necessary)
+- sensor:
   - platform: systemmonitor
     resources:
       - type: disk_use_percent
         arg: /config
+      - type: disk_use
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.